.travis.yml

```diff
@@ -1,8 +1,25 @@
 language: php
 php:
- - '5.4'
- - '5.5'
- - '5.6'
- - '7.0'
- - hhvm
- - nightly
+  - 5.6
+  - 7.0
+  - 7.1
+  - hhvm
+  - nightly
+
+matrix:
+  allow_failures:
+    - php: 5.6
+    - php: 7.0
```

Comment:
Why the `php-5.6` and `php-7.0` allow to be failure?

Reply:
Take a look at the code that is currently failing under those versions. It is beening tested, just failing. I have gone to far into changing the code logic.

Reply:
@techno-express, it doesn't make sense for these PHP versions will be failed because they're stable/normal PHP versions.

I think we should check why they're failed.

Reply:
7.0 is actually passing, 5.6 is giving parse ':' error line 139, in DNSQuery

Reply:
It seems i added that return type, will undo.